docs/Deployment.md

```diff
@@ -14,17 +14,21 @@ After signing is configured, it is possible to then configure deployment to Goog
 ## Automated Deployment
 Automated deployment to Google Play configured with the [Gradle Play Publisher plugin](https://github.com/Triple-T/gradle-play-publisher).
 To perform a deployment:
-1. Configure a Google Cloud service API key with the correct permissions
+1. Configure a Google Cloud service account and API key with the correct permissions
 1. Configure Gradle properties
+    1. `ZCASH_GOOGLE_PLAY_SERVICE_ACCOUNT` - Set the Google Play Service Account enabled in the Google Cloud console
```

Comment:
```suggestion
    1. `ZCASH_GOOGLE_PLAY_SERVICE_ACCOUNT` - Enable the Google Play Service Account in the Google Cloud console
```

Reply:
This is correct wording, as we want user to set the Google Play Service Account enabled in the Google Cloud console to this gradle field.